src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const rentCar = 40;
+
+  if (days >= 7) {
+    result = rentCar * days - 50;
```

Comment:
so here you will just return like this
```suggestion
    return baseRent - PRO_DISCOUNT;
```